packs/tfe_fdo_nomad/README.md

```diff
@@ -0,0 +1,162 @@
+  ```bash
+  $ nomad acl policy apply \
+   -namespace terraform-enterprise -job tfe-job \
+   -group tfe-group -task tfe-task \
+   terraform-enterprise-policy ./terraform_enterprise_policy.hcl
+  ``` 
+
+4. Create the necessary Nomad Variables for each job. 
+  
+  These contain sensitive data that are required like certs, licenses and passwords.
+  Create a variable specification file: 
+
+  ```hcl
+  # spec.nv.hcl
+
+  # Path where Nomad variables will be stored, the same path will be used inside TFE job file rendered by tfe.nomad.tpl for TFE job to access.
+  path      = "nomad/jobs/tfe-job"
+  namespace = "terraform-enterprise"
+
+  items {
+    # TFE DB password. Mapped to the TFE_DB_PASSWORD environment variable.
+    db_password = ""
+
+    # The field should contain the base64 encoded value of the cert. Mappped to the TFE_TLS_CERT_FILE environment variable.
+    cert = ""
+
+    # The field should contain the base64 encoded value of the bundle. Mapped to the TFE_TLS_CA_BUNDLE_FILE environment variable.
+    bundle = ""
+
+    # The field should contain the base64 encoded value of the key. Mappped to the TFE_TLS_KEY_FILE environment variable.
+    key = ""
+  
+    # A valid TFE license. Mapped to the TFE_LICENSE environment variable.
+    tfe_license = ""
+    
+    # Object storage access key. Mapped to the TFE_OBJECT_STORAGE_S3_SECRET_ACCESS_KEY environment variable.
+    s3_secret_key = ""
+
+    # The field should contain the base64 encoded value of the Nomad CA. Mapped to the TFE_RUN_PIPELINE_NOMAD_TLS_CONFIG_CA_CERT environment variable.
+    nomad_ca_cert = ""
+
+    # The field should contain the base64 encoded value of the Nomad cert. Mapped to the TFE_RUN_PIPELINE_NOMAD_TLS_CONFIG_CLIENT_CERT environment variable.
+    nomad_cert = ""
+
+    # The field should contain the base64 encoded value of the Nomad cert's key. Mapped to the TFE_RUN_PIPELINE_NOMAD_TLS_CONFIG_CLIENT_KEY environment variable.
+    nomad_cert_key = ""
+
+    # TFE Redis password. Mapped to the TFE_REDIS_PASSWORD environment variable.
+    redis_password = ""
+
+    # TFE Vault encryption key. Mapped to the TFE_ENCRYPTION_PASSWORD environment variable.
+    tfe_encryption_password = ""
+
+    # Password for the registry where the TFE image is hosted. Mapped to the TFE_IMAGE_REGISTRY_PASSWORD environment variable.
+    tfe_image_registry_password = ""
+
+  }
+  ```
+  
+  Update the `path` variable if default value of `job_name` is overridden in the `var.hcl` file.
+  The variables can be created as below by passing the `spec.nv.hcl` file we create above:
+
+  ```bash
+  $ nomad var put @spec.nv.hcl
+  ```
+
+
+<!-- Include information about how to use your pack -->
+
+# Pack Information
+
+After completing prerequisites, the pack can be run using the following command:
+```bash
+$nomad-pack run tfe_fdo_nomad -f var.hcl
+```
+
+The `var.hcl` file should contain the necessary variables required for the pack to run. The variables are listed below.
```

Comment:
This file should be `variables.hcl` to be consistent with other Packs, and because that's what the file below is actually named.